CONTRIBUTING.md

```diff
@@ -374,11 +374,38 @@ Follow these steps when updating JUnit 5:
 
 ### Others
 
-Pioneer does not take on run-time dependencies outside of JUnit 5.
-Of course test dependencies like AssertJ and build dependencies on Gradle plugins are fair game.
-To keep them updated, run `gradle dependencyUpdates`, which lists all dependencies for which a newer version exists.
-Updates still need to be done manually.
+JUnit Pioneer handles dependencies beyond JUnit 5 differently depending on how they impact its users.
 
+#### For Execution
+
+Pioneer avoids adding to users' dependency hell and hence doesn't take on dependencies beyond JUnit 5 that are _required_ at run time.
+_Optional_ dependencies are acceptable if they are needed to provide specific features, particularly:
+
+* to _integrate_ with other tools, frameworks, and libraries by offering features that directly interact with them (a hypothetical example is [Playwright](https://playwright.dev/))
+* for _ease of use_ when recreating functionality would be too complex or otherwise out of scope for Pioneer (a hypothetical example is Jackson for JSON parsing)
```

Comment:
```suggestion
* to _integrate_ with other tools, frameworks, and libraries by offering features that directly interact with them (a hypothetical example is [Playwright](https://playwright.dev) for E2E testing)
* for _ease of use_ when recreating functionality would be too complex or otherwise out of scope for Pioneer (a hypothetical example is [Jackson](https://github.com/FasterXML/jackson) for JSON parsing)
```